juriscraper/pacer/email.py

```diff
@@ -45,3 +52,71 @@ def _get_date_filed(self):
         return convert_date_string(
             date_filed[0].lower().replace("filed on ", "")
         )
+
+    def _get_document_number(self):
+        document_number = self._get_document_number_from_link()
+        return (
+            document_number
+            if document_number is not None
+            else self._get_document_number_from_text()
+        )
+
+    def _get_document_number_from_link(self):
+        try:
+            path = '//td[contains(., "Document Number:")]/following-sibling::td[1]/a/text()'
+            return numbers_only(self.tree.xpath(path)[0])[0]
+        except IndexError:
+            return None
+
+    def _get_document_number_from_text(self):
+        try:
+            path = '//td[contains(., "Document Number:")]/following-sibling::td[1]/text()'
+            return numbers_only(self.tree.xpath(path)[0])[0]
+        except IndexError:
+            return None
+
+    def _get_document_url(self):
+        try:
+            path = '//td[contains(., "Document Number:")]/following-sibling::td[1]/a/@href'
+            return self.tree.xpath(path)[0]
+        except IndexError:
+            return None
+
+    def _get_docket_entries(self):
+        try:
+            path = '//strong[contains(., "Docket Text:")]/following-sibling::font[1]/b/text()'
+            description = clean_string(self.tree.xpath(path)[0])
+            return [
+                {
+                    "date_filed": self._get_date_filed(),
+                    "description": description,
+                    "document_number": self._get_document_number(),
+                    "document_url": self._get_document_url(),
+                    "pacer_doc_id": None,
+                    "pacer_seq_no": None,
+                }
+            ]
```

Comment:
I think it's possible for there to be more than one docket entry, isn't it? Do you need an example of one like that?

Reply:
Yes, I do need an example (if you can get one). This is something else I would like to discuss in today's meeting, just understanding the docket terminology would be helpful here.

Reply:
This doesn' thave to be a blocker. We'll get to it in time.